x/auth/client/cli/tx_sign.go

```diff
@@ -29,11 +29,10 @@ func GetSignBatchCommand() *cobra.Command {
 		Short: "Sign transaction batch files",
 		Long: `Sign batch files of transactions generated with --generate-only.
 The command processes list of transactions from file (one StdTx each line), generate
-signed transactions or signatures and print their JSON encoding, delimited by '\n'.
-As the signatures are generated, the command updates the sequence number accordingly.
+signed transactions print their JSON encoding, delimited by '\n'.
```

Comment:
```suggestion
signed transactions or signatures and print their JSON encoding, delimited by '\n'.
```

I think `or signatures` is useful - it applies when `--signature-only` is used.

Reply:
ok

Reply:
There is still a typo here: `signed transactions print their JSON encoding, delimited by '\n'.`